### PR TITLE
Summarizer sample: Format markdown output

### PR DESCRIPTION
**Pull request by sebastianbenz**
Also call the API in the same way as the official sample.

### CODE REVIEW

File: functional-samples/ai.gemini-on-device-summarization/sidepanel/index.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+import DOMPurify from '../node_modules/dompurify/dist/purify.es.mjs';
+import { marked } from '../node_modules/marked/marked.min';
```
Comment (oliverdunk) [inline functional-samples/ai.gemini-on-device-summarization/sidepanel/index.js:2]: It's quite unusual to import from a relative path like this. Could you look at what setup would be needed so you can just import from "marked"?

**Review by oliverdunk — APPROVED** [review]
This generally looks good to me! Love the use of dompurify.

**Comment by sebastianbenz** [comment]
New look:

Screenshot 2024-11-08 at 21 18 39

@oliverdunk PTAL 
@tomayac PTAL

File: functional-samples/ai.gemini-on-device-summarization/sidepanel/index.js (hunk 1 above)
Comment (sebastianbenz) [reply functional-samples/ai.gemini-on-device-summarization/sidepanel/index.js]: fixed

File: functional-samples/ai.gemini-on-device-summarization/sidepanel/index.js [hunk 2]
```diff
@@ -55,39 +78,30 @@ async function generateSummary(text) {
...
     throw new Error('AI Summarization is not supported in this browser');
   }
   const canSummarize = await window.ai.summarizer.capabilities();
   if (canSummarize.available === 'no') {
-    throw new Error('AI Summarization is not availabe');
+    throw new Error('AI Summarization is not supported');
   }
-
-  const summarizationSession = await window.ai.summarizer.create();
+  const summarizationSession = await window.ai.summarizer.create(
```
Comment (tomayac) [inline functional-samples/ai.gemini-on-device-summarization/sidepanel/index.js:89]:
```suggestion
  const summarizationSession = await self.ai.summarizer.create(
```

Shorter and worker-compatible.

File: functional-samples/ai.gemini-on-device-summarization/manifest.json [hunk 3]
```diff
@@ -1,8 +1,12 @@
 {
-  "name": "Summarizer",
+  "name": "Summarization API sample",
   "version": "0.1",
   "manifest_version": 3,
-  "description": "Get a short summary of every webpage.",
+  "description": "Get a short summary of every webpage using Chrome's built-in summarization API.",
```
Comment (tomayac) [inline functional-samples/ai.gemini-on-device-summarization/manifest.json:5]:
```suggestion
  "description": "Get a short summary of webpage content using Chrome's built-in summarization API.",
```

"Every" would be a lie, given the context window constraints.

**Review by tomayac — APPROVED** [review]
LGTM with two mega nits.

**Comment by sebastianbenz** [comment]
Thanks Tom!

**Pull request merged**